lib/install.sh

```diff
@@ -148,6 +148,9 @@ map_python_packages()
 		centos-[9]*)
 			map_python2_to_python3
 			;;
+		redhat-[9]*)
```

Comment:
is it possible to check detect-system.sh, so we can set _system_name="Rocky", but i'm not sure whether it contains file /etc/os-release or /etc/redhat-release, and what is the content.

If it can be named as Rocky, help add a soft link rockylinux-9 in distro-adaptation to point to centos-9.